WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/details/BackupDownloadDetailsFragment.kt

```diff
@@ -74,6 +74,17 @@ class BackupDownloadDetailsFragment : Fragment() {
         ((recycler_view.adapter) as BackupDownloadDetailsAdapter).update(content.items)
     }
 
+    // todo: annmarie -(REMOVE) this dummy method references layout files that lint says aren't
+    // used, but they will be in the next PR because there were too many changes for 1 PR
+    private fun dummyAccess() {
```

Comment:
@ParaskP7 - I created and committed all the layout files and submitted with the PR. The PR would fail CI without this fun and it is gone in the very next PR. The dummy fun passes CI - believe you me, I wasn't happy that I this was all committed, but by the time I noticed I was two branches past this one.

Reply:
Yeap, it is always tricky to work with chained branches and PRs. 😞 

I have left an elaborate comment on your [1st PR in this group(https://github.com/wordpress-mobile/WordPress-Android/pull/13586). Let me know your thoughts.